src/style.css

```diff
@@ -0,0 +1,115 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+body,
+h3,
+p {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.card {
```

Comment:
[Checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) the 11th point

Reply:
I know this, I did not complete the checklist and therefore did not change the status of the task in the excel file